lib/ddtrace/ext/ci.rb

```diff
@@ -63,24 +63,32 @@ def filter_sensitive_info(url)
 
       def extract_appveyor(env)
         url = "https://ci.appveyor.com/project/#{env['APPVEYOR_REPO_NAME']}/builds/#{env['APPVEYOR_BUILD_ID']}"
+
+        if !env['APPVEYOR_REPO_PROVIDER'].nil? && env['APPVEYOR_REPO_PROVIDER'] == 'github'
+          repository = "https://github.com/#{env['APPVEYOR_REPO_NAME']}.git"
+          commit = env['APPVEYOR_REPO_COMMIT']
+          branch = (env['APPVEYOR_PULL_REQUEST_HEAD_REPO_BRANCH'] || env['APPVEYOR_REPO_BRANCH'])
+          tag = env['APPVEYOR_REPO_TAG_NAME']
+        end
+
         {
           TAG_PROVIDER_NAME => 'appveyor',
-          Git::TAG_REPOSITORY_URL =>  "https://github.com/#{env['APPVEYOR_REPO_NAME']}.git",
-          Git::TAG_COMMIT_SHA => env['APPVEYOR_REPO_COMMIT'],
+          Git::TAG_REPOSITORY_URL =>  repository,
```

Comment:
If the `if` statement above is not executed, do we expect `Git::TAG_REPOSITORY_URL` to be empty?

Reply:
Yes, we're supporting only Github as SCM tool atm, but you're right. We will iterate this point in the future for sure.